backend/coreapp/tests.py

```diff
@@ -229,7 +225,12 @@ def test_mwcc_wine(self):
         """
         result = CompilerWrapper.compile_code("mwcc_247_92", "-str reuse -inline on -fp off -O0", "int func(void) { return 5; }", "extern char libvar1;\r\nextern char libvar2;\r\n")
 
-        self.assertGreater(len(result.elf_object), 0, "The compilation result should be non-null")
+    def test_dummy_compiler(self):
+        """
+        Ensure basic functionality works for the dummy compiler
+        """
+
+        result = CompilerWrapper.compile_code("dummy", "", "sample text 123", "")
 
```

Comment:
is no assertion required?

Reply:
If the compilation returns a non-200, the test automatically fails per the test framework

Reply:
or, sorry. this isn't an api call. if the function raises an error (which it will if there are errors), the test will fail *